Composer/packages/types/src/orchestrator.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type IOrchestratorModelRequest = {
```

Comment:
nit: please remove the I prefix, that's the convention for defining interfaces in other language such as C#, Java, etc. not JS

Reply:
with the current type, change IOrchestratorModelRequest to OrchestratorModelRequest (removing the prefix I)